customer-docs/quickstart.md

```diff
@@ -340,4 +340,128 @@ make first-run-all
 
 You can now go preview data in the staging tables:
 
-![Preview Data](../images/quickstart-preview-data.png)
+![Preview Data](./images/quickstart-preview-data.png)
+
+# Alternate Option: 
```

Comment:
I do not like quickstarts which are "choose your own" adventure because it's hard to maintain and confusing for newbies. From a phData perspective, we don't recommend the SQS only method.

Let's update to only have one method, the new one.